test/release/examples/benchmarks/shootout/binarytrees.chpl

```diff
@@ -20,14 +20,15 @@ proc main() {
   //
   // Create the "stretch" tree, checksum it, print its stats, and free it.
   //
-  const strTree = new unmanaged Tree(strDepth);
-  writeln("stretch tree of depth ", strDepth, "\t check: ", strTree.sum());
-  delete strTree;
+  {
```

Comment:
The new scope isn't an obvious win to me over the manual delete, but I'm not opposed if you prefer it. Does the comment above need to be updated? Technically it will still be freed, but there's no longer an explicit line to associate that comment with

Reply:
I prefer it slightly from the delete-free perspective.  I agree that the text of the comment could be clarified a bit...